engine/src/main/java/org/destinationsol/events/ForceEvent.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.events;
+
+import com.badlogic.gdx.math.Vector2;
+import org.terasology.gestalt.entitysystem.event.Event;
+
+/**
+ * Event that represents a continuous force applied to an entity, like gravity. This does NOT represent anything
+ * that is a sudden, instantaneous force, like a collision. That type of occurrence is handled by {@link ImpulseEvent}.
+ */
+public class ForceEvent implements Event {
```

Comment:
Because a force is applied to an object over a period of time, the lack of anything related to time is a tad confusing?

How would one use this event? Would you send it and the force is just applied forever? Do you send the event constantly until you want the force to end?

Reply:
Force events are sent repeatedly each timestep, for the duration of the time that the entity is being affected by the force.